src/PKSim.Infrastructure/Serialization/Xml/Serializers/ApplicationSettingsXmlSerializer.cs

```diff
@@ -14,6 +14,7 @@ public ApplicationSettingsXmlSerializer()
 
       public override void PerformMapping()
       {
+         Map(x => x.MoBiPath);
```

Comment:
Ok, I thought if they might be zipped together, PKSim could also check a sort of 'parallel' folder for MoBi.

If they aren't distributed that way, then it's not possible.